### ISSUE TITLE
t2 restore bricks tessel

### ISSUE THREAD

**Issue by dbuentello** [tessel/t2-cli#824]
**Summary:** After rebooting from  `t2 restore`, the tessel will have a steady green led and fading in/out blue led. `t2 list` will not detect the board but you CAN still run `t2 restore` (still reboots to bricked state)

**Workaround:** reflash to firmware _0.0.12_ using [flash.py](https://github.com/tessel/t2-firmware/blob/master/scripts/flash.py) followed by `t2 update`

---
 
* Using OS X _10.10.5_ and node _v4.4.3_
* `t2 restore`
```
INFO Looking for your Tessel...
INFO Connected to your Tessel.
INFO Downloading files. This may take a few minutes...
Downloading [====================] 100% 0.0s remaining
INFO Download complete!
INFO Restoring your Tessel...
INFO Bulk Erasing Flash Memory...
INFO Writing U-Boot...
INFO Writing MediaTek factory partition...
INFO Writing OpenWRT SquashFS (this may take a few minutes)...
INFO Restore successful. Please reboot your Tessel
```
* `t2 list --usb --loglevel=debug`
```
INFO Searching for nearby Tessels...
DEBUG (discovery) Will scan for USB devices
DEBUG (discovery:usb) Device found.
DEBUG (discovery) Error opening device. Error: LIBUSB_ERROR_PIPE
WARN Detected a Tessel that may be booting.
DEBUG (discovery) Timeout hit! Waiting for pending to finish
DEBUG (discovery) Done! Stopping.
WARN No Tessels Found.
```
* Used workaround to restore tessel to operational state
* Repeated `t2 restore` on Linux machine (node _v4.5.0_)
* Tessel rebooted to bricked state 
* `t2 list --usb --loglevel=debug`
```
INFO Searching for nearby Tessels...
DEBUG (discovery) Will scan for LAN devices
DEBUG (discovery) Will scan for USB devices
DEBUG (discovery:usb) Device found.
DEBUG (discovery) Connection opened: undefined
DEBUG (discovery) Fetching name: undefined
DEBUG (commands:usb) uci,get,system.@system[0].hostname
DEBUG (usb-process) Opening process for: ucigetsystem.@system[0].hostname
ERR! Detected CLI crash { [Error: LIBUSB_TRANSFER_ERROR] errno: 1 } Error: LIBUSB_TRANSFER_ERROR
ERR!     at Error (native)
INFO Crash Reported: http://crash-reporter.tessel.io/crashes?fingerprint=0x1032aff9
DEBUG (discovery) Timeout hit! Waiting for pending to finish
```

**Comment by dbuentello** [comment]
- Checking out cbd6b3fc85395b729a1641fd3eea848d56b3d1e9 also fixes issue

**Comment by HipsterBrown** [comment]
This issue has been fixed in the release of [v0.0.26](https://github.com/tessel/t2-cli/releases/tag/v0.0.26).
